x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -12132,7 +12130,7 @@
     "xpack.ml.newJob.wizard.jobDetailsStep.advancedSection.enableModelPlot.title": "モデルプロットを有効にする",
     "xpack.ml.newJob.wizard.jobDetailsStep.advancedSection.enableModelPlotAnnotations.description": "選択すると、モデルが大幅に変更されたときに注釈を生成します。たとえば、ステップが変更されると、期間や傾向が検出されます。",
     "xpack.ml.newJob.wizard.jobDetailsStep.advancedSection.enableModelPlotAnnotations.title": "モデル変更注釈を有効にする",
-    "xpack.ml.newJob.wizard.jobDetailsStep.advancedSection.mmlWarning.message": "モデルプロットの作成には大量のリソースを消費する可能性があり、選択されたフィールドの基数が100を超える場合はお勧めしません。このジョブの予測基数は{highCardinality}です。 この構成でモデルプロットを有効にする場合、専用の結果インデックスを選択することをお勧めします。",
+    "xpack.ml.newJob.wizard.jobDetailsStep.advancedSection.mmlWarning.message": "モデルプロットの作成には大量のリソースを消費する可能性があり、選択されたフィールドの基数が100を超える場合はお勧めしません。このジョブの予測基数は{highCardinality}です。 この構成でモデルプロットを有効にする場合、専用の結果インデックスを選択することをお勧めします。",
```

Comment:
I'm not sure why there's a diff here. I think we should try to leave any translations not related to your changes as-is.

Reply:
Will do, thanks.